### DISCUSSION TITLE
I can't move the camera on pokémon snap

### DISCUSSION THREAD

**Discussion by guiidzeroeight** [mupen64plus-ae/mupen64plus-ae#1038] [General]
I like the emulator,but I found one problem,not on the emulator itself,but on a game,in Pokemon snao you just can't move the camera!please fix this!

**Comment by fzurita** [comment]
Hi, for that game, I recommend the parallel emulation profile if your phone
supports vulkan, or the software renderer, otherwise you will have such
issues.
